README.md

```diff
@@ -31,4 +31,4 @@ This project is licensed under Apache License, Version 2.0 ([LICENSE][] or http:
   [crates]: https://img.shields.io/crates/v/portgraph
   [LICENSE]: LICENCE
   [msrv]: https://img.shields.io/badge/rust-1.70.0%2B-blue.svg?maxAge=3600
-  [RELEASES]: RELEASES.rst
\ No newline at end of file
+  [CHANGELOG]: CHANGELOG.md
```

Comment:
I note this used to refer to RELEASES.rst, but there doesn't look to have been one - are you fixing the extension, or is something else going on?

Reply:
It was a broken link, yes